$NetBSD: patch-ab,v 1.2 2015/12/29 23:34:43 dholland Exp $

Disable shared library. "It's unversioned and quite primitive, and
nothing is currently depending on it."

--- configure.orig	2001-07-11 02:00:52.000000000 -0400
+++ configure
@@ -2877,8 +2877,8 @@ rm -f conftest*
     EXE=''
     if test "$GCC" = yes
     then
-      SHARED_LIB='libawka.so'
-      SHAREDFLAG='-fPIC'
+      SHARED_LIB=''
+      SHAREDFLAG=''
     fi
   else
     EXE=$djgpp
